src/Analyzer/Passes/QueryAnalysisPass.cpp

```diff
@@ -5615,9 +5615,13 @@ ProjectionNames QueryAnalyzer::resolveFunction(QueryTreeNodePtr & node, Identifi
             /// Replace storage with values storage of insertion block
             if (StoragePtr storage = scope.context->getViewSource())
             {
-                if (auto * query_node = in_second_argument->as<QueryNode>())
+                QueryTreeNodePtr table_expression;
+                /// Don't even ask... there's turtles all the way down...
```

Comment:
Better to add a useful comment that explains why this is necessary. In the future nobody will care about turtles